invenio_app_rdm/administration/user_moderation/user_moderation.py

```diff
@@ -41,43 +43,43 @@ class UserModerationListView(AdminResourceListView):
     item_field_list = {
         # custom display of the values - only declared to create columns
         "expanded.topic.user": {
-            "text": "User",
+            "text": _("User"),
```

Comment:
is this for developers or users? with users i mean every person using the browser.

Reply:
It looks like the end user to me, no?
![Screenshot 2025-03-14 at 19 19 35](https://github.com/user-attachments/assets/5f841f12-bab8-4bb3-961b-e0ffc3e6ea4c)